domains/dev101.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "An easy payment API",
+  "repo": "https://infrapay.github.io/website/",
+  "owner": {
+    "username": "InfraPay",
+    "email": "not_public@np.org",
```

Comment:
if you don't want to share your email, leave `""` as your email and share more of your social medias. we require at least 1 valid social media link beside github.